app/views/ReportView/components/GenomicSummary/index.tsx

```diff
@@ -13,21 +12,26 @@ import { useSnackbar } from 'notistack';
 import sortBy from 'lodash.sortby';
 
 import api, { ApiCallSet } from '@/services/api';
-import { getMicrobial } from '@/services/reports/microbial';
-import { getComparators } from '@/services/reports/comparators';
-import { getMutationSignatures } from '@/services/reports/mutation-signature';
-import { getMutationBurden } from '@/services/reports/mutation-burden';
 import { formatDate } from '@/utils/date';
 import EditContext from '@/components/EditContext';
 import ConfirmContext from '@/components/ConfirmContext';
-import AlterationsService from '@/services/reports/genomic-alterations.service';
 import ReadOnlyTextField from '@/components/ReadOnlyTextField';
 import DescriptionList from '@/components/DescriptionList';
 import ReportContext from '@/components/ReportContext';
 import VariantChips from './components/VariantChips';
 import VariantCounts from './components/VariantCounts';
 import PatientEdit from './components/PatientEdit';
 import TumourSummaryEdit from './components/TumourSummaryEdit';
+import {
+  MsiType,
+  PatientInformationType,
+  GeneVariantType,
+  TumourSummaryType,
+  MicrobialType,
+} from './types.d';
+import { MutationBurdenType, ComparatorType } from '../MutationBurden/types.d';
+import MutationSignatureType from '../MutationSignatures/types.d';
+import { ImmuneType } from '../Immune/types';
```

Comment:
weird how this works, but `.d` is required of all the other types ...

Reply:
I've removed the `.d` extension now